pudb/debugger.py

```diff
@@ -342,6 +344,24 @@ def set_frame_index(self, index):
 
         self.ui.stack_list._w.set_focus(self.ui.translate_ui_stack_index(index))
 
+    def open_file_to_edit(self, filename, line_number):
+
+        if not filename:
+            raise ValueError("Could not get filename as it was None")
```

Comment:
Make this more actionable for a user, explaining what may have happened.

Reply:
Actually as I see this can not happen.
I removed the raising of the exception

Reply:
DONE